participants.tsv

```diff
@@ -258,4 +258,4 @@ sub-vuiisIngenia02	F	36	2019-02-20	vuiisIngenia	Vanderbilt University Institute
 sub-vuiisIngenia03	F	25	2019-02-20	vuiisIngenia	Vanderbilt University Institute of Imaging Science	Philips	Ingenia	16-ch neurovascular	Release 5.5.0.0	"K.P. O'Grady, A.J.E. Combes, S.A. Smith"
 sub-vuiisIngenia04	F	30	2019-02-20	vuiisIngenia	Vanderbilt University Institute of Imaging Science	Philips	Ingenia	16-ch neurovascular	Release 5.5.0.0	"K.P. O'Grady, A.J.E. Combes, S.A. Smith"
 sub-vuiisIngenia05	M	29	2019-02-25	vuiisIngenia	Vanderbilt University Institute of Imaging Science	Philips	Ingenia	16-ch neurovascular	Release 5.5.0.0	"K.P. O'Grady, A.J.E. Combes, S.A. Smith"
-sub-vuiisIngenia06	M	31	2019-02-26	vuiisIngenia	Vanderbilt University Institute of Imaging Science	Philips	Ingenia	16-ch neurovascular	Release 5.5.0.0	"K.P. O'Grady, A.J.E. Combes, S.A. Smith"
\ No newline at end of file
+sub-vuiisIngenia06	M	31	2019-02-26	vuiisIngenia	Vanderbilt University Institute of Imaging Science	Philips	Ingenia	16-ch neurovascular	Release 5.5.0.0	"K.P. O'Grady, A.J.E. Combes, S.A. Smith"

```

Comment:
be careful with the added line

Reply:
the "newline" character at the end of the file has been added somehow automatically, I will restore this line to original one.

Reply:
what software are you using?

Reply:
So far, I edited the `participants.tsv` file directly on GitHub